### PR TITLE
And justice for all (greytides) - Making guncrafting easier and faster

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

TLDR Kat's weapon changes have made improv guns completely useless. Too hard to make for little payoff, the biggest joke being the slow firing, 13 damage .32 pistol. This PR remediates the issues not by removing these guns, but making them easier to craft (as it should be). This PR also reorganizes a few of our modular recipes to fit in with the new categories.

This PR is not bobweapons, this is simply repairing broken but existing content.

I will not list every single change i did to recipes, just look at the file changes and comment on them instead please. I don't need to write everything out in bold letters.

## Why It's Good For The Game

The .32 pistol is a travesty, this might make it worthwhile for at least entertainment. I hate to say it, but Kat seemingly only made the improvised weapons update to benefit engineer mains, fucking with everyone that doesn't have plentiful access to materials.

## Changelog
:cl:
balance: Makeshift weapons are now makeshift and don't cost ridiculous amounts of materials to be crafted.
balance: The improvised shotgun no longer has a damage penalty at all.
/:cl:

### CODE REVIEW

**Comment by Owai-Seek** [comment]
I'm noticing you renamed something to durathread string, but it's made out of cloth, not durathread.

**Comment by BobJoga** [comment]
> I'm noticing you renamed something to durathread string, but it's made out of cloth, not durathread.

Citadel renamed it to durathread for some reason. I might roll it back.

**Comment by ThatGuyFromThatPlace** [comment]
Wait... The .32 pistol does 13 damage? That is less than a welder/pickaxe/circular-saw...
